dapps/marketplace/src/pages/create-listing/ChooseCategory.js

```diff
@@ -125,17 +133,19 @@ const ChooseListingCategory = ({ listing, prev, next, onChange }) => {
             </div>
           </div>
         </div>
-        <div className="col-md-4">
-          <div className="gray-box">
-            <DownloadApp />
+        {!isMobileApp && (
```

Comment:
I preferred this over checking inside of `DownloadApp` because we'll probably soon render something different when `isMobileApp`.

Reply:
`isMobileApp` will be true for all mobile wallets that are built using React Native, right?

https://github.com/OriginProtocol/origin/blob/master/dapps/marketplace/src/hoc/withIsMobile.js#L36

In that case, we should use `walletType === 'Mobile' || walletType === 'Origin Wallet'` unless we don't want this in other mobile DApp browsers/wallets.